test/suite/test_checkpoint04.py

```diff
@@ -129,6 +129,9 @@ def test_checkpoint_stats(self):
                 break
             else:
                 multiplier += 1
+                # Reopen the checkpoint to reset statistics. 
+                # We don't want stats from earlier runs to interfere with later runs.
+                self.reopen_conn()
```

Comment:
I think it should be "Reopen the connection .....".

Reply:
Thanks, fixed in [f8ffacb](https://github.com/wiredtiger/wiredtiger/pull/9078/commits/f8ffacb08bc57924a14e87cff32df1a00f3e4a2b)